### DISCUSSION TITLE
How can mark a selection of my screen and copy it to the clip board

### DISCUSSION THREAD

**Discussion by lakruzz** [kea-dev/technology-intro#6] [Q&A]
GitHub issues supports that I can paste an image from the clip board directly into an issues or discussion topic.

### Use Case
If I see an error in the GUI and want to add an image of it to an issue I'd like to have a way to mark that bit of the screen that's relevant and with a single operation or key-combination copy it to the clip board.

I'm on a Mac Book but I bet Windows PC users want to have a solution to this too.

**Comment by tohasim** [comment]
You can do this by copying an image file and then either right click -> paste, or press _ctrl + v_

![image](https://user-images.githubusercontent.com/42994990/215455074-ff2c78b2-28dd-4c46-992f-975e5be02a72.png)
